patterns/2-structured/discover-your-innersource.md

```diff
@@ -48,18 +31,20 @@ People don't bother looking for internally developed solutions - they might not
 
 Make it easy to find the reusable code.
 
-* Pull in Repo names, descriptions and README.md files into the search engine assuming that a search engine exists. Note: a one-stop-shop kind of search engine for all relevant communication and documentation is difficult to attain. Even when using Github's enterprise offering, people often deploy additional systems like wikis to host content orthogonal to what is in the code repositories, slack channels (or IRC), mailing lists (or even nntp servers), some place to store stuff written down in office formats, search over personal e-mail etc. Several of these systems come with search built-in, but integrating this content in one search box or even just deploying a federated search engine across all sources often doesn't come off-the-shelf.
+* Pull in repo names, descriptions and `README.md` files into the search engine assuming that a search engine exists.
+  * A one-stop-shop kind of search engine for all relevant communication and documentation is difficult to attain.
+  * Even when using GitHub's enterprise offering, people often deploy additional systems like wikis to host content orthogonal to what is in the code repositories, slack channels (or IRC), mailing lists (or even nntp servers), some place to store stuff written down in office formats, search over personal e-mail etc.
+  * Several of these systems come with search built-in, but integrating this content in one search box or even just deploying a federated search engine across all sources often doesn't come off-the-shelf.
 * Implement process change to first check for internal solutions for this problem.
 * Tool with a central view (but people are more inclined to google externally than look internally)
```

Comment:
What does this mean?

Reply:
I think what was meant here was to create a tool that combines search results from external (google) and internal systems.

At least that would fit to this bullet in the Resulting context:
> Search results are combined (internal and external)